From a72f6b7dbb2d64c19feb6bf6702a85f86cd6eda8 Mon Sep 17 00:00:00 2001
From: "Dr. Greg" <greg@enjellic.com>
Date: Sat, 6 Feb 2021 10:09:50 -0600
Subject: [PATCH 10/50] Convert documentation to refer to events rather then
 behavior.

This is a non-functionality changing patch that corrects
documentation to refer to modifying the configuration of the
events rather then behavior namespace.
---
 security/integrity/ima/ima_events.c | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/security/integrity/ima/ima_events.c b/security/integrity/ima/ima_events.c
index 3b2a755a0570..84fd4e9cd4c6 100644
--- a/security/integrity/ima/ima_events.c
+++ b/security/integrity/ima/ima_events.c
@@ -126,7 +126,7 @@ struct AIevent AI_events[] = {
 };
 
 
-/* Structure defining a behavior contour point. */
+/* Structure defining an event contour point. */
 struct contour {
 	struct list_head list;
 	char identity[WP256_DIGEST_SIZE];
@@ -168,7 +168,7 @@ struct event {
 
 /*
  * The following structure encapsulates all of the information needed
- * to support a behavior namespace.
+ * to support an events namespace.
  */
 struct ima_events_map {
 	char hostid[WP256_DIGEST_SIZE];
@@ -222,9 +222,9 @@ static atomic_t map_opencount = ATOMIC_INIT(1);
 
 /**
  *	current_map - This function returns the currently active
- *	behavior behavior namespace.
+ *	events namespace.
  *
- *	Returns a a pointer to the currently active behavioral map.
+ *	Returns a a pointer to the currently active evants map.
  */
 
 static struct ima_events_map *current_map(void)
@@ -235,9 +235,9 @@ static struct ima_events_map *current_map(void)
 }
 
 /**
- *	emit_event - This function is responsible for nerating a
- *	userspace behavior event.
- *	@map:	The behavior namespace the event is to be generated in.
+ *	emit_event - This function is responsible for generating a
+ *	userspace visible event report.
+ *	@map:	The namespace the event is to be generated in.
  *	@event: the enumerated event type.
  *
  *	returns the status of event generation.
@@ -431,7 +431,7 @@ static struct trajectory * create_point(const char *filename,
 }
 
 /**
- *	add_trajectory_point - Add a point to the system behavior trajectory.
+ *	add_trajectory_point - Add a point to the event trajectory.
  *	@filename: The name of the event.
  *	@actor: The actor identity elements.
  *	@subject: The subject identity elements.
@@ -597,7 +597,7 @@ static int update_behavior_measurement(struct crypto_shash *tfm, char *id)
 }
 
 /**
- *	load_pseudonym - Add a pseudonym entry to the system behavior
+ *	load_pseudonym - Add a pseudonym entry to the event namespace
  *	description.
  *	@filename: The user space buffer containing the pathname of
  *	the pseudonym.  This buffer is required to have a linefeed
@@ -678,7 +678,7 @@ static int load_pseudonym(const char __user *pathbufr, size_t datalen)
 }
 
 /**
- *	load_contour - Add a contour point to the system behavior
+ *	load_contour - Add a contour point to the event namespace
  *	description.
  *	@contour: The buffer containing the contour point in ASCII
  *	hexadecimal format.
-- 
2.31.1

